includes/Checker/CLI_Runner.php

```diff
@@ -103,4 +103,24 @@ protected function get_include_experimental_param() {
 
 		return false;
 	}
+
+	/**
+	 * Returns an array of categories for filtering the checks.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array An array of categories.
+	 */
+	protected function get_categories_param() {
+		$categories = array();
+
+		foreach ( $_SERVER['argv'] as $value ) {
+			if ( false !== strpos( $value, '--categories=' ) ) {
+				$categories = explode( ',', str_replace( '--categories=', '', $value ) );
```

Comment:
You use `wp_parse_list()` above, so the same function should be used here as well instead of manual `explode()`.